About.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+
```

Comment:
- [ ] Based on the project requirement, you are expected to deploy your project using the GitHub Pages and then add the link to the deployed page to your README Demo section. 
![image](https://github.com/umarahgul/capstone-1/assets/84156714/a02358b1-867d-49a5-9da4-4ba999179809)